internal/storage/caching/fast_stat_bucket.go

```diff
@@ -83,6 +85,36 @@ func (b *fastStatBucket) insertMultiple(objs []*gcs.Object) {
 	}
 }
 
+func (b *fastStatBucket) insertListing(listing *gcs.Listing) {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+
+	expiration := b.clock.Now().Add(b.ttl)
+
+	// save the objects in cache
```

Comment:
nit: Start with capital later and end with full stop.

Reply:
done